src/components/App.js

```diff
@@ -1,14 +1,105 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
```

Comment:
It's always a good practice to fix the lint errors instead of disabling them. Please fix these errors.